mysql-5-6.yml

```diff
@@ -0,0 +1,5 @@
+version: "3.1"
```

Comment:
does one need `version` in these partial files?

Reply:
Yep, otherwise it will implicitly add `version: "3.9"`